README.md

```diff
@@ -1 +1,29 @@
-Modeling Workflow Engine (MWE)
\ No newline at end of file
+# Modeling Workflow Engine (MWE)
+
+## Release Process
+
+MWE releases are performed via the master branch on [Jenkins](https://ci.eclipse.org/mwe/job/mwe2/job/master/). The job is parameterized so the release type ('Integration', 'Beta', 'M1', 'M2', 'M3', 'RC1', 'RC2', 'GA') can be configured upon starting the job.
+
+### Nightly Integration Releases
+
+The master branch is configured to perform a nightly integration build. If a nightly build fails, running the build manually choosing the 'Integration' release type should return the same result and further actions should not be necessary.
+
+### Stable Releases
+
+Performing a stable release has a few manual steps that are documented here:
+
+1. Start the master branch job, selecting the release type. When the job is finished, artifacts should have been uploaded to the Sonartype Nexus and the Eclipse download storage.
+
+1. Check Jenkins artifacts, if maven artifacts have the correct versions (without qualifiers!).
+
+1. Check Jenkins artifacts, if p2 artifacts have the correct version (with qualifiers!).
+
+1. Check if the [MWE download site](https://www.eclipse.org/modeling/emft/downloads/index.php?project=mwe&showAll=1&showMax=5&sortBy=) now provides the correct artifacts.
+
+1. Check if the [MWE update site](https://download.eclipse.org/modeling/emft/mwe/updates) now provides the correct artifacts.
+
+1. Check if Sonartype Nexus provides the correct artifacts. If so, close the staging repository and release it afterwards.
```

Comment:
Sonartype -> Sonatype
Add link

Reply:
fixed typo, but i think one link when mentioning nexus the first time should be enough?!

Reply:
I'm fine with that